app/scripts/modules/core/src/serverGroup/serverGroup.transformer.js

```diff
@@ -2,16 +2,27 @@
 
 const angular = require('angular');
 
-import { PROVIDER_SERVICE_DELEGATE } from 'core/cloudProvider/providerService.delegate';
+import { ACCOUNT_SERVICE, PROVIDER_SERVICE_DELEGATE } from '@spinnaker/core';
```

Comment:
Inside the `core` module, you should still use the `core/cloudProvider/...` style import.